packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/paypal-commerce-alternative-method-ratepay-payment-strategy.spec.ts

```diff
@@ -0,0 +1,188 @@
+import {
+    BillingAddress,
+    InvalidArgumentError,
+    OrderFinalizationNotRequiredError,
+    PaymentArgumentInvalidError,
+    PaymentInitializeOptions,
+    PaymentIntegrationService,
+    PaymentMethod,
+    PaymentMethodInvalidError,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+import { LoadingIndicator } from '@bigcommerce/checkout-sdk/ui';
+import PayPalCommerceIntegrationService from '../paypal-commerce-integration-service';
+import { PayPalCommerceHostWindow, PayPalSDK } from '../paypal-commerce-types';
+import PayPalCommerceAlternativeMethodRatePayPaymentStrategy from './paypal-commerce-alternative-method-ratepay-payment-strategy';
+import { getPayPalCommerceIntegrationServiceMock } from '../mocks';
+import { PaymentIntegrationServiceMock } from '@bigcommerce/checkout-sdk/payment-integrations-test-utils';
+import { PaypalCommerceRatePay } from './paypal-commerce-alternative-methods-payment-initialize-options';
+import getPayPalCommerceRatePayPaymentMethod from '../mocks/get-paypal-commerce-ratepay-payment-method.mock';
+
+describe('PayPalCommerceAlternativeMethodRatePayPaymentStrategy', () => {
```

Comment:
Here might be a lots of test due to the functionality that you have in the strategy. May I ask you to make ratepay strategy covered with test at least 85%-90%?

Reply:
added tests